_docs/concepts/volumes.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: guide
+title: StorageOS Docs - Volumes
+anchor: concepts
+module: concepts/volumes
+---
+
+# Volumes
+
+StorageOS volumes are a logical construct which represent a writeable volume
+and exhibit standard POSIX semantics. We present volumes as mounts into
+containers via the StorageOS FUSE layer.
+
+Conceptually, StorageOS volumes have a front end presentation, which is the
+side the application sees, and a backend presentation, which is the actual
+on-disk format. Depending on the configuration, frontend and backend components
+may be on the same or different hosts.
+
+Volumes are formatted using the linux standard ext4 filesystem as standard.
+Different filesystems may be supported in the future.
```

Comment:
Its worth checking with Alex C, but I think we support different FS. I've seen the storage class in k8s options being different than ext4. I've never tried though.

Reply:
Alex C told me yesterday "for the storageos volumes themselves, they can be ext2, ext3, ext4, xfs, btrfs."

Reply:
We do support multiple filesystems but our cli and gui options for selecting them are non existent and I don't want to muddy the waters for GA.